mephisto/client/review_server.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from flask import Flask, Blueprint, send_file, jsonify, request
+import os
+import atexit
+import signal
+import csv
+import sys
+import time
+import threading
+
+
+if False:  # sys.stdout.isatty():
+    pass
+else:
+    # disable all logging out to stdout
+    import logging
+
+    log = logging.getLogger("werkzeug")
```

Comment:
What logger is this?

Reply:
This is the suggested way to disable stdout logging for flask: https://stackoverflow.com/a/18379764

Since this util outputs the review data on stdout, I needed std out to be clean.
The name comes from an underlying lib that flask uses: https://werkzeug.palletsprojects.com/en/1.0.x/

Reply:
It could make sense to require the review file name as an argument instead? I'm unsure anyone would specifically need the stdout setting.

Reply:
Good idea! I think that the file name will definitely be used a lot more. If anything I think we can move the cutesy stdout setting behind a flag instead of default behavior.